Python/specialize.c

```diff
@@ -270,6 +270,15 @@ print_optimization_stats(FILE *out, OptimizationStats *stats)
             );
         }
     }
+
+    for (int i = 0; i < 512; i++){
```

Comment:
`0` is reserved and is the initial value of `lastuop` and 512 should be replaced with `MAX_UOP_ID+1`
```suggestion
    for (int i = 1; i <= MAX_UOP_ID; i++){
```